app/tikzktexteditorview.cpp

```diff
@@ -54,12 +53,8 @@ TikzKTextEditorView::TikzKTextEditorView(QWidget *parent) : TikzEditorViewAbstra
     connect(m_currentDoc, &KTextEditor::Document::urlChanged, this,
             &TikzKTextEditorView::documentUrlChanged);
 
-    KTextEditor::CodeCompletionInterface *cci =
-            qobject_cast<KTextEditor::CodeCompletionInterface *>(m_documentView);
-    if (cci) {
-        m_completion = new TikzKTextEditorCompletion(this);
-        cci->registerCompletionModel(m_completion);
-    }
+    m_completion = new TikzKTextEditorCompletion(this);
+    m_documentView->registerCompletionModel(m_completion);
```

Comment:
I'm not sure if this is at fault, but I tried writing some code and completion doesn't seem to be working.